src/App/Components/Form.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-unused-vars */
+import React from 'react';
+import Select from 'react-select';
+
+const Form = () => {
+  const bookCategories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+  const options = bookCategories.map((category) => ({
+    value: category,
+    label: category,
+  }));
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log('e');
```

Comment:
- [ ]  [REQUIRED] No console.log committed to repo. It's ugly, it kills performance. and it can make confidential data be visible in the browser tools. (screenshot from Pathwright)